app/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamLabOrderSender.java

```diff
@@ -85,7 +87,11 @@ protected String requestToken() {
         try {
             senderToken =
                     jwt.generateSenderToken(
-                            CLIENT_NAME, RS_DOMAIN_NAME, "pemKey", CLIENT_NAME, 300);
+                            CLIENT_NAME,
+                            RS_DOMAIN_NAME,
+                            secrets.getKey("AZURE_KEY_NAME"),
```

Comment:
Azure key name should go here.

Reply:
Here's what to put in replacement of `"AZURE_KEY_NAME"`.

```java
var senderPrivateKey = "report-stream-sender-private-key-" + ApplicationContext.getEnvironment();
secrets.getKey(senderPrivateKey)
```